File: Mathematical-PROB/Quadratic_Equation.cpp
#include<bits/stdc++.h>
using namespace std;

int countDis(int a, int b, int c){

    return ((b*b)-(4*a*c));
}

int main(){

    int a,b,c;
    cout << "Enter a : ";
    cin >> a;
    cout << "Enter b : ";
    cin >> b;
    cout << "Enter c : ";
    cin >> c;

    int dis = countDis(a,b,c);

    if(dis > 0){

        int X1 = (-b+sqrt(dis))/(2*a);
        int X2 = (-b-sqrt(dis))/(2*a);

        cout << X1 << " " << X2;
    }
    else if(dis == 0){

        int X = -b/(2*a);
        cout << X << " is an unique root";
    }
    else{
        cout << "The roots are not real!";
    }
}

File: Mathematical-PROB/QuadraticEquation.java
import java.util.Scanner;

public class QuadraticEquation{

    public static void main(String[] args) {
        
        int a,b,c;
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a : ");
        a = input.nextInt();
        System.out.print("Enter b : ");
        b = input.nextInt();
        System.out.print("Enter c : ");
        c = input.nextInt();

        int d = (b*b)-(4*a*c);

        if(d >0){

            double X1 = (-b +Math.sqrt(d))/(2*a);
            double X2 = (-b -Math.sqrt(d))/(2*a);

            System.out.println(X1 + " " + X2);
        }
        else if( d == 0){

            int X = -b/(2*a);
            System.out.println(X + " is an unique root");
        }
        else{
            System.err.println("This roots are not real");
        }

        input.close();
    }
}